### PR TITLE
fix: hotfix version 1.3.1 for Android

### PR DESCRIPTION
**Pull request by uziasferreirazup**
Hotfix merge into master for version 1.3.1 Android

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: android/beagle/src/main/java/br/com/zup/beagle/android/utils/ActionExtensions.kt [hunk 1]
```diff
@@ -23,11 +23,13 @@ import br.com.zup.beagle.android.context.ContextActionExecutor
 import br.com.zup.beagle.android.context.ContextData
 import br.com.zup.beagle.android.context.expressionOf
 import br.com.zup.beagle.android.context.hasExpression
+import br.com.zup.beagle.android.context.tokenizer.function.builtin.toJSONArray
```
Comment (botbeaglezup) [inline android/beagle/src/main/java/br/com/zup/beagle/android/utils/ActionExtensions.kt:26]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/aaca07ca879d29f5cfb594ca1e146994f05e074a/android/beagle/src/main/java/br/com/zup/beagle/android/utils/ActionExtensions.kt#L26"></span>The import ‘br.com.zup.beagle.android.context.tokenizer.function.builtin.toJSONArray’ is unused.</td>
    </tr>
  </tbody>
</table>

File: android/beagle/src/main/java/br/com/zup/beagle/android/utils/ActionExtensions.kt [hunk 2]
```diff
@@ -23,11 +23,13 @@ import br.com.zup.beagle.android.context.ContextActionExecutor
...
 import br.com.zup.beagle.android.logger.BeagleMessageLogs
 import br.com.zup.beagle.android.utils.HandleEventDeprecatedConstants.HANDLE_EVENT_ACTIONS_POINTER
 import br.com.zup.beagle.android.utils.HandleEventDeprecatedConstants.HANDLE_EVENT_DEPRECATED_MESSAGE
 import br.com.zup.beagle.android.utils.HandleEventDeprecatedConstants.HANDLE_EVENT_POINTER
 import br.com.zup.beagle.android.widget.RootView
+import java.lang.reflect.Type
```
Comment (botbeaglezup) [inline android/beagle/src/main/java/br/com/zup/beagle/android/utils/ActionExtensions.kt:32]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/aaca07ca879d29f5cfb594ca1e146994f05e074a/android/beagle/src/main/java/br/com/zup/beagle/android/utils/ActionExtensions.kt#L32"></span>The import ‘java.lang.reflect.Type’ is unused.</td>
    </tr>
  </tbody>
</table>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/aaca07ca879d29f5cfb594ca1e146994f05e074a/android/beagle/src/main/java/br/com/zup/beagle/android/view/BeagleFragment.kt#L19">android/beagle/src/main/java/br/com/zup/beagle/android/view/BeagleFragment.kt#L19</a> - The import ‘android.content.Context’ is unused.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by tuliopereirazup — APPROVED** [review]

**Review by matheusribeirozup — APPROVED** [review]

**Pull request merged**